tests/unit/pretender-query-param-handler-test.js

```diff
@@ -562,4 +562,34 @@ module('pretender-query-params-handler', function () {
       );
     });
   });
+
+  module('query params for handlerFound', function (hooks) {
+    hooks.beforeEach(function () {
+      this.server = new QueryParamAwarePretender(() => {}, {
+        forcePassthrough: true,
+      });
+    });
+
+    hooks.afterEach(function () {
+      this.server.shutdown();
+    });
+
+    test('query params are used to find the handler', async function (assert) {
+      assert.expect(1);
+
+      this.server.get('/api/graphql?foo=bar', (request) => {
+        assert.deepEqual(
+          request.queryParams,
+          {
+            foo: 'bar',
+          },
+          'query params do exist on the request'
+        );
```

Comment:
Prior to the changes in `_handlerFor`, how did this fail? What did `request.queryParams` have on it?

Reply:
Prior to this change `request.queryParams` had no fields